index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>ITMO landing</title>
+        <link rel="stylesheet" href="./pages/index.css" />
+    </head>
+    <body>
+        <div class="root">
+            <header class="header"></header>
+            <main class="content">
+                <section class="intro"></section>
+                <section class="about"></section>
+                <section class="lab"></section>
+                <section class="projects"></section>
+                <section class="education"></section>
+                <section class="articles"></section>
+                <section class="team"></section>
+                <section class="partners"></section>
+                <section class="feedback"></section>
+                <footer class="footer"></footer>
```

Comment:
There aren't any mistakes in naming, thank you! 
According to the structure rules, the footer comes after </main> and before </body>